.travis.yml

```diff
@@ -1,7 +1,6 @@
 sudo: false
 language: ruby
 rvm:
-- 2.1.9
 - 2.2.5
 - 2.3.1
```

Comment:
Can you add 2.4.2 to this?

Reply:
Will do, I just didn’t want to interfere with @DirtyF’s pull request #49 too much 😃.

Perhaps, this would be a good time to update all minor versions then?

Reply:
@pluehne  on other repos we use the following:

```
2.2
2.3
2.4
```

Let's be consistent.

Reply:
@DirtyF: Thanks for the suggestion!

For some reason, Jekyll itself [doesn’t follow that convention](https://github.com/jekyll/jekyll/blob/2b6330b6862cbd3fbad663f4a92279b1ab12deeb/.travis.yml#L8_L11), but I agree that leaving out the patch version makes sense 😃 for clarity and maintainability.

I added a commit addressing this below.

Reply:
Thanks, we'll get there too 😄